src/components/SliderItem.tsx

```diff
@@ -0,0 +1,39 @@
+import { FunctionComponent } from "react";
+export interface ISliderItemProps {
+  slides: Array<{ url: string; title: string }>;
+  buttonHandlers?: () => void;
+  currentIndex: number;
+  setCurrentIndex: (newVal: number) => void;

```

Comment:
Check if component uses all property that declared in interface?

Reply:
- [ ] Done. [b5bab08](https://github.com/AnastasiSo/Portfolio-Project/pull/16/commits/b5bab08c81dac87477aa7feb8de8c1dba78324bc).